### PR TITLE
Load 8021q module if it isn't already loaded

### PR DESCRIPTION
**Pull request by kargig**
This removes a warning when first trying to use vconfig and 8021q hasn't
been loaded yet.

### CODE REVIEW

**Comment by skalkoto** [comment]
if you want this to be included in the upcoming release (0.1), do a pull request against the release-0.1 branch.

File: prv-net-helper.in [hunk 1]
```diff
@@ -18,6 +18,9 @@ function up {
 	offset=$4
 
 	echo "Adding VLANs $2 - $3"
+	if [ ! -f /proc/net/vlan/config ]; then
+		modprobe 8021q
```
Comment (skalkoto) [inline prv-net-helper.in]: `modprobe` should probably be considered an external executable and be autodetected in ./configure and be overwritable be the `MODPROBE` variable like the rest of the external executables.

**Comment by skalkoto** [comment]
LGTM

Do we want this to land to 0.1rc2 or 0.2?

**Comment by kargig** [comment]
The fix is quite minor, I don't think it matters that much which "release" it's going to go into. Do as you please.

**Comment by skalkoto** [comment]
commits landed to release-0.1 branch

**Pull request closed without merging**